consumer_kafka/src/main/java/com/example/Application.java

```diff
@@ -23,53 +23,35 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.kafka.ConcurrentKafkaListenerContainerFactoryConfigurer;
 import org.springframework.context.annotation.Bean;
 import org.springframework.kafka.annotation.KafkaListener;
-import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
-import org.springframework.kafka.core.ConsumerFactory;
-import org.springframework.kafka.core.KafkaTemplate;
-import org.springframework.kafka.listener.DeadLetterPublishingRecoverer;
-import org.springframework.kafka.listener.SeekToCurrentErrorHandler;
 import org.springframework.kafka.support.converter.RecordMessageConverter;
 import org.springframework.kafka.support.converter.StringJsonMessageConverter;
-import org.springframework.util.backoff.BackOff;
-import org.springframework.util.backoff.FixedBackOff;
+import org.springframework.messaging.Message;
 
 /**
- *
  * @author Gary Russell
+ * @author Chris Bono
  * @since 2.2.1
- *
  */
 @SpringBootApplication
 public class Application {
 
 	private final Logger logger = LoggerFactory.getLogger(Application.class);
 
+	Foo2 storedFoo;
+
+	Message<Foo2> storedFooMessage;
+
 	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 	}
 
-	@Bean
-	public ConcurrentKafkaListenerContainerFactory<?, ?> kafkaListenerContainerFactory(
```

Comment:
@marcingrzejszczak I removed the DLT aspect of this as
1. It was not used
2. w/o it removes the producer config from the application.yml which makes it more clear what needs to be set by the consumer application.